Building is broken for Svelte

`npm run build` does not fail, but the resulting build references a stylesheet `App.css.js` that doesn't exist – so the page doesn't actually show when served.

Attempting to use the webpack plugin fails for a similar reason:

```
▼ Console

  [log] ! optimizing dependencies...
  [log]
    ⦿ web_modules/           size       gzip       brotli
      └─ svelte/internal.js  51.67 KB   13.02 KB   11.26 KB


  [error]
  [error] ./_dist_/App.js
  Module not found: Error: Can't resolve './App.css.js' in '/Users/nickisnoble/Code/new-dir/.build/_dist_'
  resolve './App.css.js' in '/Users/nickisnoble/Code/new-dir/.build/_dist_'
    using description file: /Users/nickisnoble/Code/new-dir/package.json (relative path: ./.build/_dist_)
      Field 'browser' doesn't contain a valid alias configuration
      using description file: /Users/nickisnoble/Code/new-dir/package.json (relative path: ./.build/_dist_/App.css.js)
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js doesn't exist
        .wasm
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.wasm doesn't exist
        .mjs
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.mjs doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.js doesn't exist
        .json
          Field 'browser' doesn't contain a valid alias configuration
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.json doesn't exist
        as directory
          /Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js doesn't exist
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.wasm]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.mjs]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.js]
  [/Users/nickisnoble/Code/new-dir/.build/_dist_/App.css.js.json]
   @ ./_dist_/App.js 1:0-22
   @ ./_dist_/index.js
  [log]  5 assets

▶ Build Complete!
```

The above is all from a fresh run of CSA, with no other modification past these two steps (let me know if I missed something):

1. `npm install @snowpack/plugin-webpack -D`
2. Update snowpack config:
```
{
  "extends": "@snowpack/app-scripts-svelte",
  "scripts": {},
  "plugins": ["@snowpack/plugin-webpack"]
}
```

--- a/src/commands/import-resolver.ts
+++ b/src/commands/import-resolver.ts
@@ -31,7 +31,7 @@ function resolveSourceSpecifier(spec: string, stats: Stats | false, isBundled: b
   if (stats && stats.isDirectory()) {
     const trailingSlash = spec.endsWith('/') ? '' : '/';
     spec = spec + trailingSlash + 'index.js';
-  } else if (!stats && !spec.endsWith('.js')) {
+  } else if (!stats && !spec.endsWith('.js') && !spec.endsWith('.css')) {
     spec = spec + '.js';
   }
   const {baseExt} = getExt(spec);

